RELEASE_NOTES.md

```diff
@@ -37,6 +37,60 @@ You'll need to add `tailwind.config.js` to your app `.gitignore`. If you generat
 
 You can read more about this change on PR [\#9480](https://github.com/decidim/decidim/pull/9480).
 
+### 3.2 Change Webpacker to Shakapacker
+
+Since the Rails team has retired the Webpacker in favour or importmap-rails or js-bundling, we got ouserlves in a situation where performance improvements could not be performed.
+In order to continue having support for Webpacker like syntax, we have switched to Shakapacker.
+
+In order to perform the update, you will need to make sure that you do not have webpacker in your Gemfile.
+If you have it, please remove it, or change it to
+
+```ruby
+gem "shakapacker", "~> 6.5"
```

Comment:
Why do we require instance developers to do this?

Isn't it already a dependency in the `decidim-core` gem meaning it should be automatically installed when they are depending on `decidim-core`?

Reply:
I am not requiring the developers to ADD it to their gemfile, but to CHANGE it to shakapacker if they have Webpacker in their dependencies. This is a step to avoid ending up in a situation when an instance would have shakapacker and webpacker installed in the same time.